patches/squirrel.mac/build_add_gn_config.patch

```diff
@@ -0,0 +1,764 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Samuel Attard <samuel.r.attard@gmail.com>
+Date: Thu, 25 Jun 2020 14:39:52 -0700
+Subject: build: add GN config
+
+Adds BUILD.gn file and upgrades from ReactiveCocoa to ReactiveObjC
+
+diff --git a/.gitignore b/.gitignore
+index 89c499e451ecb48655cfd42b01ffa1da56998c2e..98f80aad43a87ed75ca1660ad6a178db9a644a3a 100644
+--- a/.gitignore
++++ b/.gitignore
+@@ -1,6 +1,5 @@
+ # Xcode
+ .DS_Store
+-build/
+ *.pbxuser
+ !default.pbxuser
+ *.mode1v3
+@@ -16,3 +15,4 @@ profile
+ *.moved-aside
+ DerivedData
+ .idea/
++vendor
+diff --git a/BUILD.gn b/BUILD.gn
+new file mode 100644
+index 0000000000000000000000000000000000000000..7b006f5863e0d3525173bdc4167f5cf15c02159f
+--- /dev/null
++++ b/BUILD.gn
+@@ -0,0 +1,221 @@
++assert(is_mac)
++
++import("//build/config/mac/rules.gni")
++import("build/xcrun.gni")
++import("filenames.gni")
++
++bundle_data("mantle_headers") {
++  sources = []
++  public_deps = []
++  sources += mantle_filenames.headers
++  outputs = [ "{{bundle_contents_dir}}/Headers/{{source_file_part}}" ]
++}
```

Comment:
do we still need these if we're no longer bundling the headers?

Reply:
We need the headers during build time, but we don't want them when we zip up the app